### DISCUSSION TITLE
Help!Help!   many sectors are in this state

### DISCUSSION THREAD

**Discussion by Mrchengzhenhua** [filecoin-project/lotus#9087] [General Q&A]
9.	2022-07-26 16:44:52 +0800 CST:	[event;sealing.SectorProofReady]	{"User":{"Proof":"pyw9mWLnYQfuv8RZhH2awRZV3jSn2FmGJX0XuP/crnOdOKTbFyqHk6u0DddzPNV0gEV/lXdbkwE4K5R1MZIxlT0bGYfLxz/0Frb1ikdoiLfUKLJrXi16SL/Lw0toNSp0GSqejokr3RKUf8xJwRwIz5ncGvlOyK3hgwbbLjQpcZbiTd6Jvd3Sqf3UmUnrSQudigt/Oa446ICQSAPRIERJ/4a26mOP4EbYgmsgNTToas7+CNuLyQ+BKyjirke4d1I/hejTABJR4JdFbwJ8EUzxIrULKA5c2l6aEO1IxDRqKA5eAMxS7ftwiXgub8HctxWopF6U5MxbH20MznFw8cvSJ2+uHzm9BWVFjkpA2z6wID25SzokJ0VjTiyxvGFY0VfoEQrEpsMcCJX6NilhQ6JRlPT4q1RKprD2kx/08TQayBQXAuGN9mjjmOcb445R8wRsjf3TBL4OUXZH2Ntz1H7IxvOPQuA0yR69qkJ6U15CIpZvQ9LywAvSEFs0vTUcRztNuQQJpuEDZxFEzG/uH9Di5XXqgy8mvizvFPD9bjAM1f7yiVf998UGLTl720FR9LPmuMeoPfjIk25bB1onW2Lks4+LN+zFbZ90pv0/090hk+vGeWok/Di5yxfXIl34KmXFC/PjB0iJ5eJDD70JPU+QF3H76dxkh20A5MzbnZ86r4jT8mSXS717PzQw2Ehh573blliYnT8dfvGQ+288Q/nTc8VsHOvvdqrChsrsor9heajEGNih2k4sHNhLeQnydHpXs9qfR4LKHTV6twwCIVaowdhX/ag7R5RJfScE/WmpHRpNN1fbDT4AE91PmNR+hKMhtOo+M2jPZSjCu/U0+h6U/Gg6OsWDeHslWG58jq4Z+5LhrRyZOLtrVuBGJeFJK8dICWpGhF4wOWX8t+GTNOVnvJ1ldW8h0Z4A8DvmQxKHlSoJeW9RqFIzfrcBWG0uqciqluDNFIRntwRPGe6yu6UqOXOxX2yovoiqsUTeCLM7UwRyrL+Xnif826mzoEeHhZ9bgEa69f/IlFrZPhK7qYwebPZunATc+7Ul/5MeTNNfgOdTrObK9oiknc0MJwP/phu0i+++4yL0eUDfb3sqVhIluvusi3MIKLSjy3Mfvy6q1g0t88dAb03U0tc4OPf+j2U7BS3TxoTh7LqWOTiTNyzCRRyqNUpypZl/Aq1XKQyU6rL9rIa8PCBl5VHZZ9mERgbYhaQHHRUfStRjCwXHkUwaHLiEOVOBzjLqG3gV2+UouWOv+Tqzo2Vr2jWplYNYhx6ItsUSYvbUEAxctDpIiYUqv3/CIVTYqsWmMW4YIMRg4SiZ/dvEOQluRNlrjt6fN1G0rLKcWnB1wLKrW56NEjdoQDG/yAWpqGDWaGoX5kheCSnJOy5j71+/6CR/y5VbcKWlBDmIeGrWnAW6CElrvWb8QGVUmsOtJZ6qRM1eB68+Jqs+0DouJbISiMVdGQLKeRnxoq5NgFSEFRY+TBu9GKmTfzUupWp4M8qgmNdJmzzzCdPtCfr929wPGMT6sdap4ue/liBIm5qYkdEnqjJEL5Le/cOUksQ8CLD/TsYuxymhNkWZtgPJc/Jtzh9FaeHbHB7bgoxXof3Ka11KqGMTEuuSqkXNBpjNLyl+nLrqcJyUDJeyei4tcphNl/hfvD8i+1kGF7ENAVEjQHvLdNjZxmtgb/JCMXJtSosf+3FPdy39OuCvidmiUjoxQ79YiORw+j37k4ha82Gc0SO1ZWhXSa6OPH7/OKnsDaxCPzEyyA0U8APhFDjpbhfApBptqG54n9lvl3Q8AWn01G9tXRwTstGWhvCjDvqKQWTT1s9wApjyLeChVQuLpfKKKuOYNFuPCltSirazB57R1t0x0U1zVl6AjOmyBugtNUaHWrHf/JC55te1cb/0hEdSbtXB4TH4vQDICoRqvhwxdL3GxNgTmdXN9OOp6ELreEw8B9oTepTQWjCxkATDyX27HkHbqkP9MX/kgRG9waQyt0wd+oS1DSwsWibdwYpUwUzkjkzeLEMcPwHkBxFMRZOO4IlnavuUN2WrsjzEY4sWvzvn00S7B5b0JB9Am0HsJ63XbrvHahCG/KI9c+Fl4jDjBodfrleiWvjnkRFtsBtx93ccKwlkKBKOtekqVH2W3MbHea1XN4BttB3WhB59HQGXXeaqg4dI6cMvCyKgBy8zH6vA5hZsNTwniy8Cmbw3qoxi6fi9/wXnXgAASXcG265qKNbvqZsIJlf9sWhJWc7KWRZQcZKoBUv2jeQL4nBgQg0q6ALYZbXE5VQ/jPSmrIOAZO4Gsca4/Rdbo4tD3H/1dQiGytCHPP47A0IjvsKRjiYO7epgfrJSBHWs5FWjvJLyomBtqUq4D9namMwSnsrx4ToaraYhfj03vbd8wqKqkFdfECNV0BSFLFksJRNh5eAKZUeEIsGWZErZDxaloH2A2rqbjWJ+/VWwJfJTx7LzFWufqqU1Gs/1G9tTaziYrNbFQ2YRhob8PfXwhQJGntZkL8hgupHK6ouIpR3G5qTU18O7GaI6/lWCBBhC441P/w0uJTIKbyqoulAQ"}}
10.	2022-07-26 16:48:09 +0800 CST:	[event;sealing.SectorFinalized]	{"User":{}}
11.	2022-07-26 16:48:11 +0800 CST:	[event;sealing.SectorCommitSubmitted]	{"User":{"Message":{"/":"bafy2bzaceakghefbmzjhlgrnp5wt7zk4qnbrnyd46ay445u4nprakv27fprze"}}}
12.	2022-07-26 17:03:01 +0800 CST:	[event;sealing.SectorCommitFailed]	{"User":{}}
	failed to wait for porep inclusion: found message with equal nonce as the one we are looking for that is NOT a valid replacement message (F:bafy2bzaceakghefbmzjhlgrnp5wt7zk4qnbrnyd46ay445u4nprakv27fprze n 129537, TS: bafy2bzaceceq5o3dx5ooaidpwhuqgbrrfflmljdurr2dsxkuzvaiiaqameqk6 n129537)
13.	2022-07-26 17:04:01 +0800 CST:	[event;sealing.SectorRetryCommitWait]	{"User":{}}

**Comment by rjan90** [comment]
Hey @Mrchengzhenhua! Your messages nonces seems to be out of order. 

At first I would suggest that you check your `lotus mpool pending --local` and check if there is any messages there. I have seen this happen before when people for example running multiple lotus-nodes that the message gets sent to, and one of the nodes is out of sync.

**Comment by rjan90** [comment]
Hey @Mrchengzhenhua! Did you fix this?
